frontend/src/pages/Home/Home.tsx

```diff
@@ -1,7 +1,35 @@
-import React from "react";
+import { ToastContainer, toast } from "react-toastify";
 
+import { Link } from "react-router-dom";
+import React from "react";
 const Home: React.FC = () => {
-  return <div className="Home">home</div>;
+  const notify = () => toast("Wow so easy !");
+
+  return (
+    <>
+      <ToastContainer />
+
+      <button onClick={notify}>Notify !</button>
```

Comment:
This is only for testing purpose so we can see the toasts in action. 
It'll allow us to notify users if somethng bad happened on the server side or the app didn't like a thing or two

Reply:
that's actually cool. Would like to see implementation of it in action.